.github/workflows/main.yml

```diff
@@ -25,7 +25,7 @@ env:
   DOTNET_VERSION: 6.0
```

Comment:
please have a look at:
- SFBaseTest line 175 and 179
- Snowflake.Data.Tests.csproj line 35
- ExternalBrowserAuthenticator line 207
- RestParams line 61

Reply:
The `#if` directives are removed for `ExternalBrowserAuthenticator` and `RestParams`

But the `Snowflake.Data.Tests` and `SFBaseTest` files need the `#if` directives since the project targets specific implementations (e.g. net462, net471, netcoreapp3.1, net5.0, …)